model/simulationtests/electric_equipment_ITE.rb

```diff
@@ -0,0 +1,168 @@
+# This tests the classes that derive from ExteriorLoadDefinition and ExteriorLoadInstance
+
+require 'openstudio'
+require 'lib/baseline_model'
+
+model = BaselineModel.new
+
+#make a 2 story, 100m X 50m, 10 zone core/perimeter building
+model.add_geometry({"length" => 100,
+              "width" => 50,
+              "num_floors" => 1,
+              "floor_to_floor_height" => 4,
+              "plenum_height" => 1,
+              "perimeter_zone_depth" => 10})
+
+#add windows at a 40% window-to-wall ratio
+model.add_windows({"wwr" => 0.4,
+                  "offset" => 1,
+                  "application_type" => "Above Floor"})
+
+#add thermostats
+model.add_thermostats({"heating_setpoint" => 15,
+                      "cooling_setpoint" => 28})
+
+#assign constructions from a local library to the walls/windows/etc. in the model
+model.set_constructions()
+
+#set whole building space type; simplified 90.1-2004 Large Office Whole Building
+model.set_space_type()
+
+#add design days to the model (Chicago)
+model.add_design_days()
+
+
+# In order to produce more consistent results between different runs,
+# we sort the zones by names
+zones = model.getThermalZones.sort_by{|z| z.name.to_s}
+
+# Get spaces, ordered by name to ensure consistency
+spaces = model.getSpaces.sort_by{|s| s.name.to_s}
+
+# TODO
+# set HVAC for the IT equipment zone
+# OpenStudio Core: Forward translator => add node from zoneHVAC as well
+model.add_hvac({"ashrae_sys_num" => '07'})
+
+
+spaces.each_with_index do |space, i|
+  # IT equipment with default curves
+  if i == 0
+    it_equipment_def = OpenStudio::Model::ElectricEquipmentITEAirCooledDefinition.new(model)
+    it_equipment_def.setName("IT equipment def 1")
+    it_equipment_def.setWattsperUnit(50000)
+    it_equipment_def.setDesignFanAirFlowRateperPowerInput(0.0001)
+    it_equipment_def.setDesignEnteringAirTemperature(22.5)    # recommended SAT 18-27C, use the middle T as design
+    it_equipment_def.setDesignFanPowerInputFraction(0.4)
+
+=begin
+    # TODO
```

Comment:
There isn't a planned date for 2.7.2, the next planned release is 2.8.0 at the end of March 2019.  This would be the release that includes the next E+ update.

Reply:
@macumber Gotcha. Just keep me updated. And let me know if you have any further questions on the ITE object. Thanks Dan!